configs/neptune.yaml

```diff
@@ -45,10 +46,13 @@ parameters:
   validate_with_map: 1
   small_annotations_size: 20
   kaggle_message: 'solution-1'
+  sampler_name: 'fixed'     # from {'fixed', 'aspect ratio'}
 
 # General parameters
   image_h: 512
   image_w: 512
+  short_dim: 400
```

Comment:
@taraspiotr I think it is better to have short_dim and long_dim next to the sample_name. I would even go as far as 

```yaml
sampler_name: 'fixed'
fixed_short_dim: 400
fixed_long_dim:600
```

To make the coupling of those parameters super clear

Reply:
@jakubczakon for 'fixed' sampler we use image_h and image_w (they are fixed height and width) and for aspect ratio we use short and long (constant aspect ratio, shorter image dimension resized to short_dim if now longer is longer than long_dim its downsized so that longer dimension == long_dim)